vector_search/utils.py

```diff
@@ -4,13 +4,35 @@
 from qdrant_client import QdrantClient, models
 
 from learning_resources.models import LearningResource
+from learning_resources.serializers import LearningResourceSerializer
 from learning_resources_search.constants import CONTENT_FILE_TYPE
 from learning_resources_search.serializers import (
     serialize_bulk_content_files,
     serialize_bulk_learning_resources,
 )
 from vector_search.encoders.utils import dense_encoder
 
+QDRANT_PARAM_MAP = {
+    "readable_id": "readable_id",
+    "resource_type": "resource_type",
+    "certification": "certification",
+    "certification_type": "certification_type.code",
+    "professional": "professional",
+    "free": "free",
+    "course_feature": "course_feature",
+    "content_feature_type": "content_feature_type",
+    "run_id": "run_id",
+    "resource_id": "resource_id",
```

Comment:
These two filters still aren't working, I believe because they have not been added to `LearningResourcesVectorSearchRequestSerializer`.  They are not available to the regular search filter either though, so I think it might be okay to either remove them here, or add them to the serializer if filtering by these might be useful.

Reply:
👍  removed them